venue/components/InviteForm.vue

```diff
@@ -50,7 +50,7 @@ export default {
   data() {
     let data = {
       referralCode: "",
-      referralLink: "https://venue.volentix.io/signup?referer=",
+      referralLink: "https://venue-dev.volentix.io/signup?referer=",
```

Comment:
Bad idea to put this in the code; it completely breaks if we push this to production.

The `og:image` below is ok because if it goes into production it doesn't break anything.